charts/nextcloud/README.md

```diff
@@ -52,7 +52,7 @@ The following table lists the configurable parameters of the nextcloud chart and
 | ------------------------------------------------------------ | -------------------------------------------------------------------------------------- | ------------------------------------------------------------ |
 | `image.repository`                                           | nextcloud Image name                                                                   | `nextcloud`                                                  |
 | `image.flavor`                                               | nextcloud Image type (Options: apache, fpm)                                            | `apache`                                                     |
-| `image.tag`                                                  | nextcloud Image tag                                                                    | `{VERSION}`                                                  |
+| `image.tag`                                                  | nextcloud Image tag                                                                    | `{VERSION}`                                                            |
```

Comment:
Not sure what's up with those random re-indentations, please remove.

Reply:
My screen isn't wide enough to fully display the table and catch all of these, will fix in a while.